tools/sgen/input/FluentUIStyle.yml

```diff
@@ -346,3 +346,17 @@ AP_MSFDrawerTokens:
   cornerRadius: $Border.radius.xxLarge
   verticalShadowOffset: $Spacing.xSmall
   horizontalShadowOffset: $Spacing.small
+
+AP_MSFPersonaGridItemTokens:
```

Comment:
Talking about the naming discussion: 

1. The term "Grid" makes me think of a matrix instead of a horizontal or vertical list of items. I think I like "Collection" or "Carousel" as you suggested before.

2. I was thinking about another name, so here's the suggestion: How about "Persona Tag" or "Persona Badge"? Not sure I like it a lot, but just throwing the idea out there.

Maybe "PersonaBadge" and "PersonaBadgeCollection" for the names?
Let me know your thoughts.

Reply:
Chewed on it a bit and yeah, I like those. I think Carousel is a bit too specific of a control, which this technically isn't.